maintenance/wikia/WikiFactoryVariables/setGlobalBasedOnWam.class.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Use this script to enable any global variable on wikias based on their WAM rank.
+ */
+require_once( __DIR__ . '/../../Maintenance.php' );
+
+class setGlobalBasedOnWam extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'limit', 'How many wikis should I change the variable for?', true, true );
+		$this->addOption( 'var', 'What variable should I set?', true, true );
+		$this->addOption( 'varValue', 'What value should I set the variable to?', true, true );
+		$this->addOption( 'dependency', 'Should I consider any other variable\'s value?', false, true );
```

Comment:
I'm not sure about naming here. Maybe we can at least add some example of usage.